### DISCUSSION TITLE
Auto trigger `vertico-quick-insert`

### DISCUSSION THREAD

**Discussion by leath-dub** [minad/vertico#508] [Q&A] [answered]
I cannot for the life of me figure out how to hook the right functions such that I can automatically trigger `vertico-quick-insert` when opening vertico. Ideally I can also only temporarily enable this when certain keymaps or for certain functions like (`consult-buffer`). Any help would be appreciated !

**Comment by minad** [comment, marked as answer]
Activating the quick selection UI immediately is not ideal since it interferes with the minibuffer filtering. I suggest to use `vertico-indexed-mode` instead.

**Reply by MirkoHernandez** [reply]
I'm trying to do something similar, call `vertico-quick-exit` after a completing-read command. I agree that the filtering+ vertico-quick is a very good combination (I mostly use this), but for some few commands (with less than 10 candidates) I think it would be very handy to allow some auto trigger of  vertico-quick.

**Reply by minad** [reply]
@MirkoHernandez The problem with auto quick for few candidates would be that behave magically differs depending on candidate number. I often start typing quickly to filter candidates, before even reading what is there, and I know that many people do that. Note that Vertico interrupts candidate updates, so the list could also lag behind. Auto quick would interfere with such usage.
